tool/webqa-process-schemaorg.js

```diff
@@ -344,6 +344,11 @@ const MANUAL_PROPERTY_CANONICAL_OVERRIDE = {
 
 };
 
+const MANUAL_TABLE_CANONICAL_OVERRIDE = {
+    'Restaurant': ['restaurant', 'diner', 'place', 'joint', 'eatery', 'canteen', 'cafeteria', 'cafe'],
+    'Hotel': ['hotel', 'resort', 'lodging', 'model', 'place']
```

Comment:
Wait, why do we have manual annotations for Hotel?

Reply:
This is only enabled when we enable manual annotation. I used this to try out the performance of hotels.

Reply:
Don't we enable manual annotation in hotels to get the manual annotation of restaurant fields?

Reply:
yes, I had a separate genie to run that. I think as the toolkit, if you say manual, you get full manual.

Reply:
That makes sense. Although you seriously need to document this, so many commands, even I have no idea what does what...

Reply:
yeah, agree. i will probably do that after emnlp